modules/gitops-synchronizing-cluster-configs-application.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * configuring-an-openshift-cluster-with-argo-cd.adoc
+
+[id="synchronizing-cluster-configs-application_{context}"]
+= Synchronizing `cluster configs` application
+
+. In the Argo CD dashboard, notice that the `cluster-configs` Argo CD application has the statuses `Missing` and `OutOfSync`. Because the application was configured with a manual sync policy, Argo CD does not sync it automatically.
+
+. Click the *Sync* button on the `cluster-configs` tile, review the changes, and then click *Synchronize*.
+
+.  Notice that the `cluster-configs` Argo CD application now has the statuses `Healthy` and `Synced`. Click the `cluster-configs` tile to check the details of the synchronized resources and their status on the cluster.
+
+. Navigate to the OpenShift web console and click *Application Launcher* and see that link to the _Red Hat Developer Blog_ is now present there. 
```

Comment:
...click the  *Application Launcher*/click <icon_screen shot> icon to see the link to _Red Hat Developer Blog_ ....